hw/bitstream/BUILD

```diff
@@ -60,7 +60,7 @@ filegroup(
         "bitstream_skip": ["skip.bit"],
         "bitstream_vivado": ["//hw/bitstream/vivado:fpga_cw310_test_rom"],
         "bitstream_gcp_splice": [":gcp_spliced_test_rom"],
-        "//conditions:default": ["@bitstreams//:bitstream_test_rom"],
+        "//conditions:default": ["@bitstreams//:chip_earlgrey_cw310_bitstream"],
```

Comment:
Should we require splicing the ROM image for this case too? Given that we plan to eliminate the separate test ROM at some point, do we want to trust the ROM that is in the bitstream cache?

(This would mean taking a hard line on requiring Vivado for working with any FPGA board.)

Reply:
Yes, and we handled it in a prior PR.